src/Opdex.Platform.WebApi/OpenApi/GetVaultProposalVotesOperationProcessor.cs

```diff
@@ -0,0 +1,38 @@
+using NSwag.Generation.Processors;
+using NSwag.Generation.Processors.Contexts;
+using System.Linq;
+
+namespace Opdex.Platform.WebApi.OpenApi;
+
+public class GetVaultProposalVotesOperationProcessor : IOperationProcessor
+{
+    public bool Process(OperationProcessorContext context)
```

Comment:
This is fancy, what's this for if its only for votes specifically?

Reply:
Meant to rename this, my bad. Was playing around with NSwag and these operation processors allow us to modify the OpenAPI schema for route params, so for instance we can add examples.

I'm still wondering whether we should apply this to the entire controller or create an operation processor for each endpoint. Probably the latter, since it's more flexible and we can tailor examples per endpoint.

Reply:
Yeah probably per endpoint, if we can and it is beneficial, maybe a shared class for each controller that the endpoint processors within inherit.